docs/contributing.md

```diff
@@ -0,0 +1,68 @@
+# Contributors guide
+
+Contributions are welcome, and they are greatly appreciated!
+Every little bit helps, and credit will always be given.
+
+## Code of conduct
+
+Please read and abide by the [Ansible Community Code of Conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html) in all interactions with the community.
+
+## Reporting bugs
+
+If you find a bug, please open an issue at <https://github.com/ansible-community/awx-operator-helm/issues>.
+
+Please also include the following information in your issue:
+
+- Your operating system name and the output of `helm show chart awx-operator/awx-operator`
+- Any details about your local setup that might be helpful in troubleshooting.
+- Detailed steps to reproduce the bug.
+
+## Writing code
+
+Want to fix a bug or implement a feature?
+Great!
```

Comment:
If you want this on separate links you need to include a blankline in Markdown:

```suggestion
Want to fix a bug or implement a feature?

Great!
```

Reply:
Thanks but I want them on the same line when rendered. I prefer semantic line breaks (one sentence per line) in plain text markup like md, rst, adoc.

Reply:
FYI > https://sembr.org/